src/constants.ts

```diff
@@ -30,3 +30,5 @@ export const ARNS_TESTNET_REGISTRY_TX =
 
 export const ARNS_DEVNET_REGISTRY_TX =
   '_NctcA2sRy1-J4OmIQZbYFPM17piNcbdBPH2ncX2RL8';
+
+export const MIO_PER_IO = 1000000;
```

Comment:
Could we change this to 1_000_000 to make it a little easier to read?

Reply:
[7ff3530](https://github.com/ar-io/ar-io-sdk/pull/85/commits/7ff35305c978f2c9defb01bb4e5fd878a7bc9813)